pyproject.toml

```diff
@@ -0,0 +1,38 @@
+[tool.poetry]
+    name = "check-python-versions"
+    version = "1.0.0"
```

Comment:
I don't want to manually keep two sets of metadata in sync.  All my current maintenance tooling supports reading/updating the version number in `src/check_python_versions/__init__.py`.

I've never used Poetry and I'm not sure I want to switch to it at this time.

Reply:
If you need it for the test suite, I suggest moving it inside tests/ somewhere.

Reply:
I put version metadata there to mimic the setup.py content, not to be used to build the package.

Into my packages, I usually have version number into `__init__.py` and into pyproject.toml, kept in sync with `commitizen` tool, FYI .

let me know if you prefer removing the version here.

Reply:
Version removed.